Assets/Scripts/BulletManagement/Bullet.cs

```diff
@@ -50,4 +50,9 @@ public virtual void SetData(Emitter emitter, BulletPattern bulletPattern, int bu
         this.bulletPattern = bulletPattern;
         this.bulletId = bulletId;
     }
+
+    public int getBulletId()
+    {
+        return this.bulletId;
+    }
```

Comment:
If we're doing this, why not just make bulletId a property? i.e. `public int BulletId { get; set; }`

Reply:
Updated, thanks.